tests/fsharp/core/printing/z.output.test.default.stderr.refemitoff.bsl

```diff
@@ -0,0 +1,340 @@
+
+
+      member this.M() = "string"
+  ----------------^
+
+stdin(764,17): error FS0438: Duplicate method. The method 'M' has the same name and signature as another method in type 'ExpectDupMethod'.
+
+
+      member this.P = "string"
+  ----------------^
+
+stdin(771,17): error FS0438: Duplicate method. The method 'get_P' has the same name and signature as another method in type 'ExpectDupProperty'.
+
+
+      type public   IBPublic   = interface inherit IAPrivate abstract Q : int end
+  ------------------^^^^^^^^
+
+stdin(778,19): error FS0410: The type 'IAPrivate' is less accessible than the value, member or type 'IBPublic' it is used in.
+
+
+      type internal IBInternal = interface inherit IAPrivate abstract Q : int end
+  ------------------^^^^^^^^^^
+
+stdin(783,19): error FS0410: The type 'IAPrivate' is less accessible than the value, member or type 'IBInternal' it is used in.
+
+
+      type public   IBPublic   = interface inherit IAInternal abstract Q : int end
+  ------------------^^^^^^^^
+
+stdin(792,19): error FS0410: The type 'IAInternal' is less accessible than the value, member or type 'IBPublic' it is used in.
+
+
+          override x.M(a:string) = 1
+  -------------------^
+
+stdin(824,20): error FS0361: The override 'M: string -> int' implements more than one abstract slot, e.g. 'abstract Regression4232.D.M: 'U -> int' and 'abstract Regression4232.D.M: 'T -> int'
+
+
+  let (|A|B|) (x:int) = A x;;
+  -----^^^^^
+
+stdin(832,6): error FS1210: Active pattern '|A|B|' has a result type containing type variables that are not determined by the input. The common cause is a when a result case is not mentioned, e.g. 'let (|A|B|) (x:int) = A x'. This can be fixed with a type constraint, e.g. 'let (|A|B|) (x:int) : Choice<int,unit> = A x'
+
+
+  let (|A|B|) (x:'a) = A x;;
+  -----^^^^^
+
+stdin(835,6): error FS1210: Active pattern '|A|B|' has a result type containing type variables that are not determined by the input. The common cause is a when a result case is not mentioned, e.g. 'let (|A|B|) (x:int) = A x'. This can be fixed with a type constraint, e.g. 'let (|A|B|) (x:int) : Choice<int,unit> = A x'
+
+
+  let (|A|B|) (p:'a) (x:int) = A p;;
+  -----^^^^^
+
+stdin(838,6): error FS1210: Active pattern '|A|B|' has a result type containing type variables that are not determined by the input. The common cause is a when a result case is not mentioned, e.g. 'let (|A|B|) (x:int) = A x'. This can be fixed with a type constraint, e.g. 'let (|A|B|) (x:int) : Choice<int,unit> = A x'
+
+
+  let (|A|B|) = failwith "" : Choice<int,int>;;
+  -----^^^^^
+
+stdin(844,6): error FS1209: Active pattern '|A|B|' is not a function
+
+
+
+stdin(0,1): warning FS2303: Accessing the internal type, method or field 'f' from a previous evaluation in F# Interactive is deprecated and may cause subsequent access errors. To enable the legacy generation of a single dynamic assembly that can access internals, use the '--refemit' option.
```

Comment:
This is a the line that gets output for the "access internals" warning.  THe rest of this file is a duplicate of the standard output from the printing tests